bolt/test/X86/dwarf4-subprogram-multiple-ranges-cus.test

```diff
@@ -0,0 +1,24 @@
+# REQUIRES: system-linux
+
+# RUN: llvm-mc -dwarf-version=4 -filetype=obj -triple x86_64-unknown-linux %p/Inputs/dwarf4-subprogram-multiple-ranges-main.s -o %t1.o
+# RUN: llvm-mc -dwarf-version=4 -filetype=obj -triple x86_64-unknown-linux %p/Inputs/dwarf4-subprogram-multiple-ranges-other.s -o %t1.o
```

Comment:
You are overriding object file.
Line below should have two object files as input.
Once that is fixed, you should see linker error, because your new inputs have exact the same function names.